src/gpodder/download.py

```diff
@@ -38,7 +38,7 @@
 
 import requests
 from requests.adapters import HTTPAdapter
-from requests.exceptions import ConnectionError, HTTPError, RequestException
+from requests.exceptions import HTTPError, RequestException
```

Comment:
why this change? requests.ConnectionError is identical to requests.exceptions.ConnectionError

Reply:
The rationale is in the commit message: Fixes flake8 error 'A004 import statement "ConnectionError" is shadowing a Python builtin'.